rsts/deployment/cluster_config/flytepropeller_config.rst

```diff
@@ -1759,7 +1759,7 @@ Co-Pilot Configuration
 delete-resource-on-finalize (bool)
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-Instructs the system to delete the resource on finalize. This ensures that no resources are kept around (potentially consuming cluster resources). This, however, will cause k8s log links to expire as soon as the resource is finalized.
+Instructs the system to delete the resource upon successful execution of a k8s pod rather than have the k8s garbage collector clean it up. This ensures that no resources are kept around (potentially consuming cluster resources). This, however, will cause k8s log links to expire as soon as the resource is finalized.
```

Comment:
This file is automatically generated by this [script](https://github.com/flyteorg/flyte/blob/master/script/generate_config_docs.sh), we should update [flyteplugin](https://github.com/flyteorg/flyteplugins/blob/d38dcd31765b2aa4c5e71c9b01aa46778641809b/go/tasks/pluginmachinery/flytek8s/config/config.go#L128) instead.

Reply:
btw, I think we could add this [thread](https://flyte-org.slack.com/archives/CP2HDHKE1/p1654440120306879) to [Q&A](https://github.com/flyteorg/flyte/discussions/categories/q-a).

Reply:
Oops, sure I will do that..